src/accumulator.jl

```diff
@@ -12,6 +12,16 @@ counter(T::Type) = Accumulator(T,Int)
 Accumulator{T,V<:Number}(dct::Dict{T,V}) = Accumulator{T,V}(copy(dct))
 counter{T}(dct::Dict{T,Int}) = Accumulator{T,Int}(copy(dct))
 
+"""
+```
```

Comment:
Might be useful to point at an example, so here is one I was just digging through:

https://github.com/JuliaLang/julia/blob/master/base/sharedarray.jl#L38-L51